signatures/rainbow/rainbow/bench/opt_Ia_skylake.yaml

```diff
@@ -0,0 +1,10 @@
+platform: Intel iXEON E3-1275 (Skylake) 3.6GHZ
+timings:
+  unit: cycles
+  gen: 9280000
+  enc|sign: 6410000
+  dec|vrfy: 3370000
```

Comment:
weird. we must have different versions maybe? I find 8.29M / 67.7K / 21.7K

Reply:
What about this?

What spec file did you use? Apparently there is really no online version of it. Mine has the following sha256sum:
`f7c399499279cd4392fcaa7ffd764677dd2c6819af3a732a2c414e6949ff3248  Rainbow/Supporting Documentation/Rainbow2round.pdf`